### PR TITLE
Reduce the days to stale a PR to 22

### PR DESCRIPTION
**Pull request by pablintino**

### CODE REVIEW

**Comment by frenzyfriday** [comment]
Sorry, 22 or 15?

**Comment by frenzyfriday** [comment]
> Sorry, 22 or 15?

ah ok, I was being dumb 🤦🏽‍♀️

File: .github/workflows/stale-pr.yml [hunk 1]
```diff
@@ -12,5 +12,7 @@ jobs:
         with:
           start-date: '2024-04-25T00:00:00Z'
```
Comment (lewisdenny) [inline .github/workflows/stale-pr.yml:13]: We should remove the start date now as well, this was added as a grace period when the stale bot was added but no longer needed.

**Review by lewisdenny — CHANGES_REQUESTED** [review]

**Comment by pablintino** [comment]
recheck

**Review by lewisdenny — APPROVED** [review]
/approve

**Pull request merged**